research/lfads/distributions.py

```diff
@@ -480,7 +480,7 @@ def __init__(self, post_zs, prior_z_process):
     logq_bxu = post_zs[0].logp(z0_bxu)
     logp_bxu = prior_z_process.logp_t(z0_bxu)
```

Comment:
@gautam1858 the loglikelihood of the first element is already calculated and added to the sum in this line and in the line above